doc/sphinxman/source/manage_release.rst

```diff
@@ -80,6 +80,18 @@ Release (e.g., ``v1.3``)
 Post-Release (e.g., ``v1.3.1``)
 -------------------------------
 
+* `Assemble postrelease changes`_
+* `Tweak Conda for postrelease`_
+* `Do final pass before release tag`_
+* `Tag postrelease`_
+* `Build Conda Psi4 stack at specific commit`_
+* `Publish to main conda label`_
+* `Build Psi4conda set`_
+* `Generate download page for psicode.org`_
+* `Publish GitHub postrelease`_
+* `Publish psicode release`_
+** `Finalize release`_
```

Comment:
Is this item supposed to be doubly-starred?

Reply:
no. I just never got around to doing it for 1.3.1 so the item didn't get polished. :-(